crates/revm/benches/bench.rs

```diff
@@ -111,7 +124,14 @@ fn bench_eval(g: &mut BenchmarkGroup<'_, WallTime>, evm: &Evm) {
         };
         let mut host = DummyHost::new(evm.env.clone());
         b.iter(|| {
-            let mut interpreter = Interpreter::new(Box::new(contract.clone()), u64::MAX, false);
+            // reset gas limit to the right amount before tx
+            evm.env.tx.gas_limit = gas_limit;
```

Comment:
Why does this need to happen?

Reply:
I remember I needed that a while ago, but now it seems to be useless. Thanks for noticing it!